### PR TITLE
Bluetooth: audio: correct arg usage in BT_CODEC_LC3_CONFIG_DATA

### PR DESCRIPTION
**Pull request by Chris-StJohn-Bose**
Correct _frame_blocks_per_sdu usage in BT_CODEC_LC3_CONFIG_DATA

Signed-off-by: Chris St John <chris_st._john@bose.com>

### CODE REVIEW

**Review by Thalley — DISMISSED** [review]

**Review by szymon-czapracki — DISMISSED** [review]

**Comment by Thalley** [comment]
@Chris-StJohn-Bose Please add a commit message to your commit :)

File: include/zephyr/bluetooth/audio/lc3.h
Comment (MariuszSkamra) [inline include/zephyr/bluetooth/audio/lc3.h:320]: Please update the comment in line 304 as well

**Review by Casper-Bonde-Bose — DISMISSED** [review]
LGTM

**Review by Thalley — DISMISSED** [review]

**Comment by Thalley** [comment]
@Chris-StJohn-Bose why did you rebase? 

Looks like no code was changed, so seems like it didn't need any rebase. 

The issue with rebasing and re-pushing is that all approvals get dismissed :D

**Comment by Chris-StJohn-Bose** [comment]
> @Chris-StJohn-Bose why did you rebase?
> 
> Looks like no code was changed, so seems like it didn't need any rebase.
> 
> The issue with rebasing and re-pushing is that all approvals get dismissed :D

@Thalley - yes it said I had 2 approvals but merge was blocked without a rebase so I rebased. How do I get this change into main?

**Comment by Thalley** [comment]
> > @Chris-StJohn-Bose why did you rebase?
> > Looks like no code was changed, so seems like it didn't need any rebase.
> > The issue with rebasing and re-pushing is that all approvals get dismissed :D
> 
> @Thalley - yes it said I had 2 approvals but merge was blocked without a rebase so I rebased. How do I get this change into main?

What do you mean by it was blocked without a rebase? 

Unless there is a file conflict, which I assume there isn't, there shouldn't be any reason to rebase :)

Once it has two approvals, then someone with merge rights will merge it at some point :)

**Review by Thalley — APPROVED** [review]

**Review by Casper-Bonde-Bose — APPROVED** [review]
LGTM

**Comment by Casper-Bonde-Bose** [comment]
@Thalley It is this message that suggest the PR needs a re-base:
image
Should that just be ignored?

**Comment by Thalley** [comment]
> @Thalley It is this message that suggest the PR needs a re-base: image Should that just be ignored?

Yes, this just indicates that `main` has newer commits than this branch (which is to be expected, as we merge roughly 10 commits per day on average).

**Comment by Thalley** [comment]
> > @Thalley It is this message that suggest the PR needs a re-base: image Should that just be ignored?
> 
> Yes, this just indicates that `main` has newer commits than this branch (which is to be expected, as we merge roughly 10 commits per day on average).

Furthermore I can also mention that pressing the `Update branch` button in the web GUI would also rebase the PR, but without dismissing the reviews, as Github does not consider this a new push.

**Comment by fabiobaltieri** [comment]
Filed https://github.com/zephyrproject-rtos/zephyr/pull/53204 to mention that button in the contribution workflow, unfortunately we can't disable it.

**Pull request merged**
